sdks/java/core/src/test/java/org/apache/beam/sdk/io/LocalFileSystemTest.java

```diff
@@ -209,6 +209,30 @@ public void testMatchWithGlob() throws Exception {
         containsInAnyOrder(expected.toArray(new String[expected.size()])));
   }
 
+  @Test
+  public void testMatchInDirectoryRelativeWithGlob() throws Exception {
```

Comment:
this test is copy pasted from `testMatchWithGlob`, can you move it to a parametrized method, so the code can be reused?

Reply:
I just simplified this test, to be more straightforward.